src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingParameterSymbol.cs

```diff
@@ -116,7 +116,16 @@ internal override ImmutableArray<byte> MarshallingDescriptor
             }
         }
 
-        internal sealed override CSharpCompilation? DeclaringCompilation // perf, not correctness
+        /// <summary>
+        /// <inheritdoc/>
+        /// </summary>
```

Comment:
```suggestion
        /// <inheritdoc/>
```

I think this is all you need, though @sharwell can correct me if I'm wrong.